ADOPTERS.md

```diff
@@ -1,6 +1,16 @@
 # Adopters
 
-This is a list of production adopters of OPA (in alphabetical order):
+<!-- Hello! If you are using OPA and contributing to this file, thank you! -->
+<!-- Please keep lines shorter than 80 characters (or so.) Links can go long. -->
```

Comment:
💭 Not anything to do in this PR.. just a thought while reading through it. I wonder if it would be worth just adding a simple lint check for this sort of thing? Would probably be nice for the doc markdown content too.